data/adapters/helpers.py

```diff
@@ -204,11 +207,64 @@ def get_ref_seq_by_spdi(spdi, species='human'):
     return seq_repo[chr_ref][start:end]
 
 
-def check_collection_loaded(collection, record_id):
+def check_collection_loaded(collection, record_id, timeout_seconds=1.0):
     try:
         db = ArangoDB().get_igvf_connection()
         col = db.collection(collection)
-        return col.has(record_id)
+        return col.has(record_id, timeout=timeout_seconds)
     except Exception as e:
         print(f'Error checking {record_id} in {collection}: {e}')
         return False
+
+
+def normalize_type(value, field_type):
+    if value in {'NaN', ''}:
+        return None
+    if field_type == 'string':
+        return value
+    elif field_type == 'int':
+        try:
+            return int(value)
+        except ValueError:
+            return None
+    elif field_type == 'boolean':
+        return value.upper() == 'TRUE'
+    elif field_type == 'list':
+        return [v.strip() for v in value.split(',') if v.strip()]
+    return value
+
+
+def parse_guide_file(filepath):
```

Comment:
are we expecting more adapters using this function (so you put in helper?), and can you put an example line here?

Reply:
MPRA sequence designs could also utilize this. Anything with standardized file formats with expected types.

Reply:
Added example lines.